modules/aioseop_opengraph.php

```diff
@@ -1816,7 +1813,7 @@ function get_all_images( $options = null, $p = null ) {
 			$images = array();
 			if ( ! empty( $image ) ) {
 				foreach ( $image as $k => $v ) {
-					$images[ $v ] = '<img height=150 src="' . $v . '">';
+					$images[ $v ] = '<img alt="" height=150 src="' . $v . '">';
```

Comment:
If you have the image ID available, you could use the image caption ( `wp_get_attachment_caption( $image_id );` ) or image title `get_the_title( $image_id )` for the alt value.

Reply:
@EkoJR That's a little beyond my skillset. If you're able to provide the code I can add it.

Reply:
@wpsmort I took a look at it, and it doesn't appear to be possible. No ID is given, and the image default to `all-in-one-seo-pack/images/default-user-image.png`, which isn't part of a site's database to get the additional data.